docs/guide/content_management.md

```diff
@@ -36,15 +36,17 @@ Under this root there are preset top level Locations in each installation which
 This part of the tree is typically used for organizing folders, articles, information pages, etc.
 This means that it contains the actual content structure of the site,
 which can be viewed by selecting the **Content structure** tab in the Content mode interface.
-The default ID number of the **Content** Location is 2; it contains a Folder Content item.
+The default ID number of the **Content** Location is 2, it can be [modified via configuration](config_repository.md#top-level-locations).
+It contains a Folder Content item.
 
 #### Media
 
 **Media** is the top level Location which stores and organizes information
 that is frequently used by Content items located below the **Content** node.
 It usually contains images, animations, documents and other files.
 They can be viewed by selecting the **Media library** tab in the Content mode interface.
-The default ID number of the **Media** Location is 43; it contains a Folder Content item.
+The default ID number of the **Media** Location is 43, it can be [modified via configuration](config_repository.md#top-level-locations).
```

Comment:
```suggestion
The default ID number of the **Media** Location is 43, but it can be [modified via configuration](config_repository.md#top-level-locations).
```